tests/foreman/api/test_contentview.py

```diff
@@ -1032,9 +893,10 @@ def test_cv_associate_rh_custom_spin(self):
         view
 
         """
-        content_view = entities.ContentView(organization=self.org.id).create()
-        content_view.set_repository_ids([self.repo.id])
-        self.assertEqual(len(content_view.read().repository), 1)
+        content_view = entities.ContentView(organization=self.org).create()
+        content_view.set_repository_ids([self.repo.id])  # pylint:disable=E1101
+        content_view = content_view.read()
```

Comment:
oops, my bad. Reviewed more accurately.
Thought about code like this:
```
        content_view = content_view.read()
        self.assertEqual(len(content_view.version), 1)
        self.assertEqual(len(content_view.puppet_module), 1)
```
and why not:
```
        self.assertEqual(len(content_view.read().version), 1)
        self.assertEqual(len(content_view.read().puppet_module), 1)
```
But I think, it is too unimportant :)